libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -709,12 +710,12 @@ public static ExpressionEvaluator MultivariateNumeric(string type, Func<IReadOnl
             VerifyExpression verify = null)
             => new ExpressionEvaluator(
                 type,
-                (expression, state) =>
+                (expression, state, options) =>
                 {
                     var result = false;
                     string error = null;
                     IReadOnlyList<object> args;
-                    (args, error) = EvaluateChildren(expression, state, verify);
+                    (args, error) = EvaluateChildren(expression, state, options, verify);
```

Comment:
>options [](start = 72, length = 7)

This is a boolean context.

Reply:
yes

Reply:
Still needs to be fixed.

---
In reply to: [400615447](https://github.com/microsoft/botbuilder-dotnet/pull/3635#discussion_r400615447) [](ancestors = 400615447)